### ISSUE TITLE
Wishlist: A function that returns the number of unlabelled graphs on n vertices

### ISSUE THREAD

**Issue by szhorvat** [igraph/igraph#2077]
**What is the feature or improvement you would like to see?**

Add a function that returns the number of unlabelled graphs on n vertices. This determines the maximum "isoclass" value, and is very useful in conjunction with motif and isoclass functions.

It's sufficient to add this for graph sizes supported by `igraph_isoclass()`.

**Use cases for the feature**

Value is often needed when working with motifs / isoclasses. E.g., if we were to implement motif counting in terms of `igraph_motifs_randesu_callback()`, we'd need this.

**References**

 - Undirected: https://oeis.org/A000088
 - Directed: https://oeis.org/A000273
 - https://mathworld.wolfram.com/SimpleGraph.html
 - The Combinatorica Book, https://www3.cs.stonybrook.edu/~skiena/combinatorica/

**Comment by szhorvat** [comment]
I propose

```
igraph_integer_t igraph_graph_count(igraph_integer_t n, igraph_bool_t directed);
```

It will support only small `n` (graph size) so that the result would fit in an `igraph_integer_t`. We can copy the OEIS tables, and be done for the foreseeable future, since the main use case is for graph sizes supported by `igraph_isoclass()` anyway.

**Comment by szhorvat** [comment]
Fixed by 485989550301ebc4cc86544183fc15aa09a6a5a6
